optional-requirements.txt

```diff
@@ -7,4 +7,5 @@ requests==2.26.0
 uvicorn[standard]==0.15.0
 scikit-learn==1.3.2
 mypy==1.9.0
-ruff==0.3.4
\ No newline at end of file
+ruff==0.3.4
```

Comment:
I think it's better to use the newest version (0.4.5). Also, I think it's better use `ruff~=0.4.5`. So it'll use any ruff >= 0.4.5 but < 0.5. See https://peps.python.org/pep-0440/#compatible-release

Reply:
Done!